server/.env

```diff
@@ -1,2 +1,4 @@
 TEAM_MEMBERS="Aecio, Bonnie, Shums, Pray, Shawn, Tongil"
 DATABASE_CONNECT="mongodb+srv://team-kiwi-user:hatchways-team-kiwi@team-kiwi-cluster-yv3im.mongodb.net/hatchways?retryWrites=true&w=majority"
+ACCESS_TOKEN_LOGIN= "7186dd676fb617a4500de62d84a71e7976a47e5f0953b2bc6c1fc65544bde024e1d782e5e73a877cc820fd6e71386a7effe255f70613e5658c54496a538dd78b"
```

Comment:
this file shouldn't be getting committed

Reply:
Please refer this.
https://www.freecodecamp.org/news/gitignore-what-is-it-and-how-to-add-to-repo/